backend/src/managers/FeedManager.ts

```diff
@@ -25,7 +26,7 @@ export default class FeedManager {
         this.redis = redis;
     }
 
-    async getSubscriptionFeed(forUserId: number, page: number, perpage: number, format: ContentFormat): Promise<PostInfo[]> {
+    async getSubscriptionFeed(forUserId: number, page: number, perpage: number, format: ContentFormat, sorting: FeedSorting): Promise<PostInfo[]> {
         const limitFrom = (page - 1) * perpage;
 
         // check if fanned out
```

Comment:
Need to add another flag, `subscriptions:${forUserId}:created_at:fanned` that corresponds to the completion of the fanning out of the new redis feed cache. The key that corresponds to the requested `sorting` method should be checked, and if it's not present, `await this.pullSubscriptionsFeed(forUserId);` should be called.